src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -34,7 +34,7 @@
  * <p>
  * Rationale: Abstract classes are convenience base class implementations of
  * interfaces, not types as such. As such they should be named to indicate this.
```

Comment:
`not types as such` -> `and are not considered types per se.`

Reply:
Done